README.md

```diff
@@ -0,0 +1 @@
+# MhdraihandzakySaragih
```

Comment:
I have been working with .NET since its first introduction. Currently I work as DevOps Architect at Microsoft + Github, I'm the founder and Community Leader of Hong Kong Microsoft Community and member of DotNetToscana. I'm speaker at technical events and I write articles for magazines. I used to be a Microsoft MVP in 2015, 2016 and 2017.

I publish a lot of technical videos about DevOps, GitHub and Azure DevOps on my YouTube Channel: DJ DZAKY REMIX

I also like to make non-technical cinematic-looking videos, take a look at my personal YouTube Channel

Connect with me: